debezium-core/src/main/java/io/debezium/relational/HistorizedRelationalSnapshotChangeEventSource.java

```diff
@@ -399,4 +399,8 @@ public boolean snapshotSchema() {
             return snapshotSchema;
         }
     }
+
+    protected Clock getClock() {
```

Comment:
Is this truly needed? I'd prefer if sub-classes stored the reference from within their own constructor, if they need it.

Reply:
@gunnarmorling I tend to disagree here - the parent class is the part of the subclass so it is a duplication. It is probably not needed to add the getter and just change the vsibility of the field.

Reply:
Ok, works for me, too.